MANAG-6,10,11: Auth, ResetPassword, vue-router

MaxibX opened the pull request.

MaxibX committed:
MANAG-6: draft of auth page

sergomyaso commented on src/components/Auth.vue:
```diff
@@ -0,0 +1,46 @@
+
+<template>
+    <div>
+        <h1>{{"Pyramidum"}}</h1>
```
Почему не просто `<h1>Pyramidum</h1>?`

sergomyaso commented on src/components/Auth.vue:
```diff
@@ -0,0 +1,46 @@
+            </button>
+            <button
+                    @click="console.log('to reset password page')">
+                {{"Забыли пароль?"}}
```
Можно просто текст, у тебя же динамики нет

MaxibX committed:
MANAG-6,10,11: Auth, ResetPassword, vue-router

Доработана страница авторизации. Добавлены страницы запроса на восстановление пароля и успешного восстановления пароля. Начало использования vue-router и Bootstrap.

sergomyaso commented on src/components/Auth.vue:
```diff
@@ -0,0 +1,80 @@
+                this.password_msg = '';
+                if (this.username == '') {
+                    this.username_msg = 'Пожалуйста, введите логин';
+                    if (this.password == '') {
```
this.username_msg = this.username === '' ? 'Пожалуйста, введите логин' : '';
this.password_msg = this.password === '' ? 'Пожалуйста, введите пароль' : '';

if (this.username !== '' && this.password !== '') {
    // Запрос к БД. Переход к пользовательскому окну
}

MaxibX committed:
validation fix
